api/container/container.py

```diff
@@ -12,17 +13,18 @@
 from service.api_logic.sports_logic import SportService
 from service.api_logic.user_logic import UserService
 from service.api_logic.news_logic import NewsService
+from service.implementation.email_sender.user_subscription_manager import UserSubscriptionManager
 from service.api_logic.games_logic import GamesService
 from service.api_logic.teams_logic import TeamsService
 
 
-
 class Container(containers.DeclarativeContainer):
     wiring_config = containers.WiringConfiguration(
         modules=[
             "api.routes.api_login",
             "api.routes.api_news",
             "api.routes.api_user_preferences",
+            "service.implementation.email_sender.user_subscription_manager",
```

Comment:
I'd move this after all API declarations (it's usually a good thing to sort names in A-Z order)